### ISSUE TITLE
Can this be adopted to not bonded N & H pairs?

### ISSUE THREAD

**Issue by yanze039** [zharmad/SpinRelax#1]
Hi,

Thanks for this contribution. But I am wondering if this code/theory can be extended to two nuclei not directly bonded? say, I have  NMR heteronuclear NOE data of N-H distant away from ~4 angstroms. (this is not a standrad protein residue, it's a noncanonical RNA nucleotide...). I have this question since when calculate `f_DD`, the `rXH` is involved, but this parameters seem have been coupled with other parameters and can't be changed trivially.

Anyway, I calculated the HetNOE value by the default parameters (with `tau` set to 10 ns) the result aligned with experiments pretty well. But I am still curious whether this is a legal calculation.

**Comment by zharmad** [comment]
If I remember correctly, the heteronuclear NOE should be a through-space effect, and should apply regardless of bonding. Please check this with textbook resources.

**Comment by yanze039** [comment]
Thanks for your reply. I agree that NOE is a spatial effect that applies to any distant nuclear. But in the context of back calculation in MD simulation, are `f_DD` or other parameters parameterized against bonded situations only?

**Comment by zharmad** [comment]
Yes but it is adjustible. As per Woessner (1962), `f_DD` seems simply to quantify the relationship between magnetic relaxation times and J(ω), where the separation vector is defined between two nuclei with a constant distance. If your non-canonical nucleotide is reasonably stable, then the distance should not vary significantly and the model would hold.

There are two places where this effective (constant) distance is set:

1. In line 164 of `spectral_densities.py` you'll find `self.rAB = 1.02e-01` nm. This defines the default distance between bonded N-H nuclei.
2. The parameter ζ defines the QM correction that increases this by default to 0.1042 nm.

If you allow the ζ to vary in the fit, e.g. via DisoS2 option in `calculate-relaxations-from-CT.py`, it would've simply adjusted the parameter such that the effective distance becomes ~0.4 nm as per your non-bonded nuclei. 

Now, if the various hetNOEs have significantly different distances, you'll probably have to modify the code to permit a re-definition of rXH for each hetNOE pair.
